expected/17.0/extra/returning.out

```diff
@@ -0,0 +1,519 @@
+DELETE FROM "type_SINT" RETURNING *;
+WARNING:  DELETE ... RETURNING doesn't return values (not implemented)
+ col 
+-----
+    
+    
+(2 rows)
+
+--Testcase 18:
+SELECT * FROM "type_BINT";
+         col          
+----------------------
+  9223372036854775807
+ -9223372036854775808
+(2 rows)
+
+--Testcase 19:
+DELETE FROM "type_BINT" RETURNING *;
+WARNING:  DELETE ... RETURNING doesn't return values (not implemented)
+ col 
+-----
+    
+    
+(2 rows)
+
+--Testcase 20:
+SELECT * FROM "type_INTEGER";
+  col   
+--------
+ 120506
+(1 row)
+
+--Testcase 21:
+DELETE FROM "type_INTEGER" RETURNING *;
+WARNING:  DELETE ... RETURNING doesn't return values (not implemented)
+ col 
+-----
+    
+(1 row)
+
+--Testcase 22:
+SELECT * FROM "type_FLOAT";
+  col   
+--------
+ 3.1415
+(1 row)
+
+--Testcase 23:
+DELETE FROM "type_FLOAT" RETURNING *;
+WARNING:  DELETE ... RETURNING doesn't return values (not implemented)
+ col 
+-----
+    
+(1 row)
+
+--Testcase 24:
+SELECT * FROM "type_DOUBLE";
+        col        
+-------------------
+          1999.012
+ 3.141592653589793
+          Infinity
+         -Infinity
+(4 rows)
+
+--Testcase 25:
+DELETE FROM "type_DOUBLE" RETURNING *;
+WARNING:  DELETE ... RETURNING doesn't return values (not implemented)
+ col 
+-----
+    
+    
+    
+    
+(4 rows)
+
+--
+set datestyle=ISO;
+--Testcase 26:
+SELECT * FROM "type_TIMESTAMP";
+           col           |          b          
+-------------------------+---------------------
+ 2017-11-06 12:34:56.789 | 2017-11-06 00:00:00
+ 2017-11-06 01:03:00     | 2017-11-07 00:00:00
+(2 rows)
+
+--Testcase 27:
+DELETE FROM "type_TIMESTAMP" RETURNING *;
+WARNING:  DELETE ... RETURNING doesn't return values (not implemented)
+ col | b 
+-----+---
+     | 
+     | 
+(2 rows)
+
+--Testcase 28:
+SELECT * FROM "type_BLOB";
+    col     
+------------
+ \xdeadbeef
+(1 row)
+
+--Testcase 29:
+DELETE FROM "type_BLOB" RETURNING *;
+WARNING:  DELETE ... RETURNING doesn't return values (not implemented)
+ col 
+-----
+ 
+(1 row)
+
+--Testcase 30:
+ALTER TABLE "type_UUID" ALTER COLUMN "i" OPTIONS (ADD key 'true');
+--Testcase 31:
+DELETE FROM "type_UUID" RETURNING *;
+WARNING:  DELETE ... RETURNING doesn't return values (not implemented)
+ i | u 
+---+---
+   | 
+   | 
+(2 rows)
+
+--Testcase 32:
+DELETE FROM "type_BITSTRING" RETURNING *;
+ERROR:  relation "type_BITSTRING" does not exist
+LINE 1: DELETE FROM "type_BITSTRING" RETURNING *;
+                    ^
+--Testcase 33:
+SELECT * FROM typetest;
+ i | v |     c      | t |            d            |           ti            
+---+---+------------+---+-------------------------+-------------------------
+ 1 | a | b          | c | 2017-11-06 12:34:56.789 | 2017-11-06 12:34:56.789
+(1 row)
+
+--Testcase 34:
+INSERT INTO "type_STRING"(col) VALUES ('string') ON CONFLICT DO NOTHING RETURNING *;
+  col   
+--------
+ string
+(1 row)
+
+--Testcase 35:
+ EXPLAIN (VERBOSE, COSTS OFF)
+INSERT INTO "type_STRING"(col) VALUES ('string') RETURNING *;
```

Comment:
Similar to https://github.com/pgspider/sqlite_fdw/pull/107#r1896620244, INSERT ... RETURNING can be pushed down to SQLite.

Reply:
Please verify `EXPLAIN` commands. is this enough in current TCs?